forum/service/src/test/java/org/exoplatform/forum/search/DiscussionSearchConnectorTestCase.java

```diff
@@ -402,7 +402,7 @@ public void testSearchDataWithDoubleQuotes() throws Exception {
     assertEquals(1, discussionSearchConnector.search(context, "\"My Topic Second\"", Collections.<String>emptyList(), 0, 5, "relevancy", "ASC").size());
     // Test with double quotes for reply
     assertEquals(2, discussionSearchConnector.search(context, "\"My Reply Abc\"", Collections.<String>emptyList(), 0, 5, "relevancy", "ASC").size());
-    assertEquals(1, discussionSearchConnector.search(context, "\"Second My\"", Collections.<String>emptyList(), 0, 5, "relevancy", "ASC").size());
+    assertEquals(2, discussionSearchConnector.search(context, "\"Second\"", Collections.<String>emptyList(), 0, 5, "relevancy", "ASC").size());
 
```

Comment:
```suggestion
assertEquals(0, discussionSearchConnector.search(context, "\"My Abc\"", Collections.<String>emptyList(), 0, 5,
            "relevancy", "ASC").size());
```
The case explained in the jira is still nok. 
I think you should ensure that this assertion is ok.